topdown/http.go

```diff
@@ -400,6 +468,26 @@ func createHTTPRequest(bctx BuiltinContext, obj ast.Object) (*http.Request, *htt
 		}
 	}
 
+	// check http client cache
+	value, found := clientCache.Get(obj)
+	if found {
+		// check if any files or env variables containing certificate/key have changed
+		modifiedFiles, err := checkIfFileModified(value.files)
+		if err != nil {
+			return nil, nil, err
+		}
+
+		if !modifiedFiles && !checkIfEnvVarModified(value.envVars) {
```

Comment:
How realistic is the scenario of changed env vars? I didn't know you could change the env vars of a running process from the outside, can you? 

I can see that you use `os.SetEnv` to update the used certs from a test, but would this be the preferred way of dealing with certs if OPA is embedded as a go-lib? (I'm wondering because that's the only way I'd see to change the env vars of a running process, from "within the process".)

Reply:
I am not sure if that's a preferred way to change certs but someone could still do it I guess 🤷‍♂️ We can remove that check if you think it's not required.

Reply:
I was just surprised to see it, because it is at odds with what I've expected. It won't do much harm, I guess. So we can keep it, I'd leave that up to you 👍